docs/user/production-considerations/task-manager-production-considerations.asciidoc

```diff
@@ -0,0 +1,140 @@
+4. If the throughput is insufficient, and {kib} instances are exhibiting high resource utilization, incrementally scale horizontally by provisioning new {kib} instances and reassess.
+
+{kib} Task Manager, like the rest of the Elastic Stack, has been designed to scale horizontally, and we recommend taking advantage of this ability to ensure mission ciritcal services such as Alerting and Reporting always have the capacity they need.
+
+Scaling horizontally requires a higher degree of coordination between {kib} instances. One way by which {kib} Task Manager coordinates with other instances is delaying its polling schedule to avoid conflicting with other instances.
+By using <<task-manager-health-monitoring>> to evaluate the <<task-manager-health-evaluate-the-runtime,date of the `last_polling_delay`>> across a deployment, we can estimate the frequency at which Task Manager resets its delay mechanism.
+A higher frequency suggests {kib} instances conflict at a high rate, which can be addressed by scaling vertically rather than horizontally, in effect reducing the required coordination.
+
+[float]
+[[task-manager-rough-throughput-estimation]]
+==== Rough Throughput Estimation
+
+Predicting the required throughout a deployment might need to support {kib} Task Management is difficult, as features can schedule an unpredictable number of tasks at a variety of scheduled cadences.
+That said, a rough lower bound can be estimated which is then used as a guide.
+
+Throughput is best thought of as a measurements in _tasks per minute_.
+
+As <<task-manager-default-scaling,mentioned above>>, a default {kib} instance can support up to `200/tpm`.
+
+Given a hypothetical deployment of 100 recurring tasks, estimating the required throughput depends entirely on their scheduled cadence.
+Suppose we expect to run 50 tasks at a cadence of `10s`, the other 50 at `20m` and in addition, we expect a couple dozen non-recurring tasks every minute.
+
+A non-recurring task requires a single execution, which means that a single {kib} instance could execute all 100 tasks in less than a minute, utilizing only half of its capacity. As these tasks are only executed once, the {kib} instance will sit idle once all tasks have executed.
+For that reason, we don't tend to include non-recurring tasks in our _tasks per minute_ calculation. Instead, we ensure a _buffer_ included in the final _lower bound_ to incurr the unpredictable cost of ad-hoc non-recurring tasks.
+
+A recurring task requires as many executions as its cadence can fit into a minute. This means that a recurring task with a `10s` schedule will require as `6/tpm`, as it will be executed 6 times per minute. On the other hand, a recurring tasks with a `20m` schedule, will only execute 3 times per hour, meaning it would only require a throughput of `0.05/tpm`, a number so small it is difficult to take it into account.
```

Comment:
```suggestion
A recurring task requires as many executions as its cadence can fit in a minute. A recurring task with a `10s` schedule will require `6/tpm`, as it will execute 6 times per minute. A recurring task with a `20m` schedule only executes 3 times per hour and only requires a throughput of `0.05/tpm`, a number so small it that is difficult to take it into account.
```